lib/Yield/PTML.cpp

```diff
@@ -701,3 +703,150 @@ std::string yield::ptml::shallowFunctionLink(std::string_view Location,
   Result.addAttribute(attributes::LocationReferences, Location);
   return Result.serialize();
 }
+
+struct CommentHelper {
+  std::string_view Indicator;
+  std::size_t LineWidth;
+  std::size_t Indentation;
+
+public:
+  /// Emit an empty comment line (an indented comment indicator).
+  std::string line() {
+    return ::ptml::Tag(tags::Div,
+                       indentationImpl(Indentation) + commentImpl(Indicator))
+             .serialize() += '\n';
+  }
+
+  /// Emit a comment line with indentation and comment indicator.
+  std::string line(std::string &&Text, std::size_t InternalIndentation = 0) {
+    revng_assert(!Text.empty());
+
+    std::string Result;
+    if (InternalIndentation == 0) {
+      // No internal indentation, so both the indicator and the text can be
+      // merged into a single tag.
+      Result = indentationImpl(Indentation)
+               + commentImpl(std::string(Indicator) + ' ' + std::move(Text));
+    } else {
+      // Internal indentation present: emit everything in its own separate tag.
+      Result = indentationImpl(Indentation) + commentImpl(Indicator)
+               + indentationImpl(InternalIndentation + 1)
+               + commentImpl(std::move(Text));
+    }
+
+    return ::ptml::Tag(tags::Div, std::move(Result)).serialize() += '\n';
+  }
+
+  std::string
+  multiline(std::string_view Text, std::string_view FirstLine = "") {
```

Comment:
This function probably makes sense as a standalone function we can reuse in the future in some public header.

Reply:
This is not a function, but an important part of this class's interface, so it'd be painful to extract just it. That being I'm Ok with disclosing the entire class if that's what you want.

Reply:
OK, I was thinking a non-comments specific string primitive similar to [Python's `textwrap`](https://docs.python.org/3/library/textwrap.html#textwrap.indent). But if it's too much of an hassle, leave it.